apps/emqx_bridge_gcp_pubsub/src/emqx_bridge_gcp_pubsub.erl

```diff
@@ -398,7 +398,9 @@ service_account_json_validator(Map) ->
             {error, #{missing_keys => MissingKeys}}
     end.
 
-service_account_json_converter(Map) when is_map(Map) ->
+service_account_json_converter(Val, #{make_serializable := true}) ->
+    Val;
+service_account_json_converter(Map, _Opts) when is_map(Map) ->
     ExpectedKeys = [
```

Comment:
@thalesmg Could you please take a look at this ? 
While the behavior of the code after changing it remains the same as before, 
we want to make sure our expected understanding is correct.

I replace all fun/1 with fun/2 in converter, because of hocon don't run converter fun when make_serializable is true( check_plain).
https://github.com/emqx/hocon/blob/master/src/hocon_tconf.erl#L1247-L1261